models/edr/data_monitoring/anomaly_detection/metrics_anomaly_score.sql

```diff
@@ -27,7 +27,11 @@ time_window_aggregation as (
         bucket_duration_hours,
         updated_at,
         avg(metric_value) over (partition by metric_name, full_table_name, column_name order by bucket_start asc rows between unbounded preceding and current row) as training_avg,
+        {% if target.type == 'clickhouse' %}
```

Comment:
We prefer to have the adapter specific code use `adapter.dispatch` and contained in the `cross_db_utils` section of the code, if you could add there an `edr_stddev` macro it would be awesome